querydsl-sql/src/main/java/com/querydsl/sql/Configuration.java

```diff
@@ -301,7 +329,7 @@ public String registerSchemaOverride(String oldSchema, String newSchema) {
      * @return previous override value
      */
     public String registerTableOverride(String oldTable, String newTable) {
-        return nameMapping.registerTableOverride(oldTable, newTable);
+        return internalNameMapping.registerTableOverride(oldTable, newTable);
```

Comment:
The _internalNameMapping_ variable contains the original (current) name mapping functionality of Configuration class. The _nameMapping_ variable contains  the original functionality if no dynamicNameMapping is set otherwise a chain that uses the original functionality and if there is no result, asks for the passed NameMapping implementation. That is why the register functions must write into the internalNameMapping (original behavior) variable. Probably the name of the variables could be slightly different not to be confusing.